newton/examples/contacts/example_hydro_pressure_slice.py

```
@@ -0,0 +1,1034 @@
+        line_starts, line_ends = build_mesh_edge_lines(self.mesh)
+        self._original_line_starts_np = line_starts
+        self._original_line_ends_np = line_ends
+        self.shape_line_starts = wp.array(line_starts, dtype=wp.vec3, device=self.device)
+        self.shape_line_ends = wp.array(line_ends, dtype=wp.vec3, device=self.device)
+
+        if hasattr(self.viewer, "set_camera"):
+            self.viewer.set_camera(wp.vec3(2.5, 2.0, 1.6), -30.0, 225.0)
+
+        if hasattr(self.viewer, "register_ui_callback"):
+            self.viewer.register_ui_callback(self.render_ui, position="side")
+        self._viser_controls: dict[str, Any] = {}
```

Ruff (0.15.10)

[error] Undefined name `Any`

(F821)